pluginDefinition.prod.json

```diff
@@ -0,0 +1,22 @@
+{
+  "identifier": "org.zowe.explorer-jes",
+  "apiVersion": "1.0.0",
+  "pluginVersion": "1.0.0",
+  "pluginType": "application",
+  "webContent": {
+    "framework": "iframe",
+    "destination": "https://${ZOWE_EXTERNAL_HOST}:${GATEWAY_PORT}/ui/v1/explorer-jes",
```

Comment:
Want to understand more when these environment variables will be replaced with real value. Will Desktop handle this?

Reply:
Thinking more about this, if pluginDefinitions.json has environment variables, and it may be changed by the admin from instance.env, then install-app.sh should not be called when the plugin is installed/configured for instance, but have to be handled at runtime configure stage.

Does this conflict with the install-app.sh idea what we show on docs website here https://docs.zowe.org/stable/extend/extend-desktop/mvd-installplugins.html?

Reply:
they will never be replaced during install, configure or start, its a template for ZLUX, which it will replace as per each user request.

Reply:
It's good news and will make the install/configure/launch scripts easier. Thanks!